ui/src/admin/containers/AdminPage.js

```diff
@@ -27,6 +41,33 @@ class AdminPage extends Component {
     }
   }
 
+  handleClickCreate(type) {
```

Comment:
Does `handleClickCreate` add the user to state or create it on the server?  Also, I feel like this would read better with a guard clause:

```
if (type !== 'users') {
  return
}

this.props.addUser()